README.md

```diff
@@ -1,30 +1,59 @@
-# Pyodide
+**[What is Pyodide?](#what-is-pyodide)** |
+**[Try Pyodide](#try-pyodide-no-installation-needed)** |
+**[Getting Started](#getting-started)** |
+**[Contributing](#contributing)** |
+**[License](#license)** |
+**[Help and Resources](#help-and-resources)**
+
+# [Pyodide](https://github.com/iodide-project/pyodide)
 
 
 [![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)
 
 The Python scientific stack, compiled to WebAssembly.
 
-It provides transparent conversion of objects between Javascript and Python.
-When inside a browser, this means Python has full access to the Web APIs.
+[**Try Pyodide and Iodide in your browser**](https://alpha.iodide.io/notebooks/300/)
+
+## What is Pyodide?
+
+**Pyodide** brings the Python runtime to the browser via WebAssembly, along with the Python scientific stack including NumPy, Pandas, Matplotlib, parts of SciPy, and NetworkX.
```

Comment:
maybe add "for a full list [look here](https://github.com/iodide-project/pyodide/tree/master/packages)"

Reply:
Great suggestion. Now I know where to find the list too. Thanks for the review.